### PR TITLE
Fixed Invalid URL

### LINKED ISSUE #26: Bug - Check for Invalid URL
The input is not checking for a valid URL, whatever the user type it considers that as an URL, so first we have to check it is a valid URL or not...

I found this bug and would like to work on it
Add hacktoberfest tag

### PR DESCRIPTION
**Pull request by Villain45**
The Check for Invalid URL is implemented
fixes #26

### CODE REVIEW

**Comment by Villain45** [comment]
@Yasshu55 any pblm? it has been 2 days still not merged

**Comment by Yasshu55** [comment]
@Villain45 You've done a great job! There are no problems everything is fine with the code from my side. Just waiting for @unnikrishnan2002 final review. He's a bit inactive due to a few reasons, If he doesn't review it by tomorrow I will merge it.

**Review by unnikrishnan2002 — APPROVED** [review]
Sorry for taking this long. LGTM 🚀

**Pull request merged**
